### DISCUSSION TITLE
Swedish translations not applied?

### DISCUSSION THREAD

**Discussion by Atque** [Stellarium/stellarium#2894] [Translations]
There might be something obvious I'm missing, but the Swedish translations, provided by me via Transifex, do not seem to apply to the beta releases. Am I doing something wrong when translating?

**Comment by alex-w** [comment]
Transifex automatic adding translation into Github when some resource reach 100% of translation.

**Reply by Atque** [reply]
I see. Is this a new policy?

**Reply by alex-w** [reply]
Yes, old API is closed

**Reply by Atque** [reply]
All right, thanks for the answer!

**Reply by Atque** [reply]
Hmmm, and only the resource 100% translated is applied?

**Comment by gzotti** [comment]
I understand if "any" resource (language) reaches 100%, this triggers an update for "All". Every few days some language reaches 100%.

**Reply by Atque** [reply]
Only one resource has been applied so far (the one I translatd as to reach 100% in any of them). Maybe the others will apply in a few hours or so? Otherwise I'll have to translate over 1000 strings to have the main program's translations tested.

**Reply by alex-w** [reply]
For example you are translated `stellarium (master)` resource in 100% and this translation will be immediately committed in main repository on GitHub

**Reply by Atque** [reply]
Okay, then there is quite a bit of work to do.
